pyfr/plugins/tavg.py

```diff
@@ -156,6 +237,7 @@ def __call__(self, intg):
         if not self._started:
             self._init_accumex(intg)
             self._started = True
+            return
```

Comment:
?

Reply:
The code should probably be refactored so that the main code path will work on the first iteration.

Reply:
Any reason to hit the main code path when we set self._started = True? Since `_init_accumex` already takes care of `self.prevex` and sets `accex` to zero.

Reply:
Seems a bit weird not to hit the main path.  Suggests init is replicating part of the main path.